mifosng-provider/src/main/java/org/mifosplatform/infrastructure/documentmanagement/api/ImagesApiResource.java

```diff
@@ -70,9 +61,9 @@ public ImagesApiResource(final PlatformSecurityContext context, final ImageReadP
     @POST
     @Consumes({ MediaType.MULTIPART_FORM_DATA })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String addNewClientImage(@PathParam("clientId") final Long clientId, @HeaderParam("Content-Length") final Long fileSize,
-            @FormDataParam("file") final InputStream inputStream, @FormDataParam("file") final FormDataContentDisposition fileDetails,
-            @FormDataParam("file") final FormDataBodyPart bodyPart) {
+    public String addNewClientImage(@PathParam("clients") final String clientName, @PathParam("clientId") final Long clientId,
```

Comment:
This method had too many parameters to begin with, but referring only to the client, it would make sense to have a Client object instead of 2 client related params passed in.